using-ide.md

```diff
@@ -187,7 +187,7 @@ To open Observatory:
 Assists are code changes related to a certain code identifier. A number of these
 are available when the cursor is placed on a Flutter widget identifier, as
 indicated by the yellow lightbulb icon. The assist can be invoked by clicking
-the lightbulb, or by using the keyboard shortcut (`Alt-Enter` on Linux and Windows, `option-return` on macOS), as illustrated
+the lightbulb, or by using the keyboard shortcut (`Alt`+`Enter` on Linux and Windows, `⌥`+`return` on macOS), as illustrated
```

Comment:
Please change to <kbd>Alt</kbd>+<kbd>Enter</kbd> on Linix .....
or by using the keyboard shortcut (<kbd>Option</kbd>+<kbd>Return</kbd> on macOS), as illustrated.  

Please don't use the lambda and other characters. That is not the style we use. We write things out.

Reply:
Oh, again, the rest of the doc was written with the symbols, I'll change those too.